### PR TITLE
2253 remove rfc again

### PR DESCRIPTION
**Pull request by gl2748**
This PR removes React-Foundation-Components dependencies from everything except the `Voting.jsx` component. It is part of a succession of PRs.

refs #2253 

Outside of task but nice to have:
- [x]  Remove React-Foundation-Components from Voting.jsx  - see: https://github.com/steemit/condenser/compare/2253-remove-rfc-again-part-2?expand=1

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 
 "@babel/code-frame@^7.0.0-beta.35":
-  version "7.0.0-beta.39"
```
Comment (bnchdrff) [inline yarn.lock]: why are there changes to yarn.lock?

File: src/app/components/modules/TopRightMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+};
+
+function TopRightMenu({
+    username,
+    showLogin,
+    logout,
+    loggedIn,
+    vertical,
+    navigate,
+    toggleOffCanvasMenu,
```
Comment (bnchdrff) [inline src/app/components/modules/TopRightMenu.jsx]: on steemitdev the menu toggle does not work -- when i click on the hamburger nothing happens

**Comment by gl2748** [comment]
Companion branch for work to remove RFC from the voting component is here:
https://github.com/steemit/condenser/compare/2253-remove-rfc-again-part-2

**Comment by bnchdrff** [comment]
please pull the voting component tests into this PR so we can track regressions for the next set of work

**Comment by bnchdrff** [comment]
(that is the requirement specified in #2253)

**Comment by gl2748** [comment]
I don't think voting mechanism tests belong in this pr. If you'd like them to be separated from this PR: 
https://github.com/steemit/condenser/pull/2817
That makes sense.
It's also worth noting that writing tests for the voting mechanism are more complex than estimated.

**Comment by bnchdrff** [comment]
are the tests done? if so let's get them into master, whether in this pr or another separate one

**Comment by gl2748** [comment]
yep all tests are here.

**Comment by gl2748** [comment]
# Changes to header component:
header_changes

# Changes to userwallet component:

userwallet_changes2

# Changes to post component:
post_changes


# Changes to userprofile component:
userprofile_changes

**Pull request merged**